flink-contrib/flink-storm-compatibility/flink-storm-compatibility-core/src/main/java/org/apache/flink/stormcompatibility/api/FlinkLocalCluster.java

```diff
@@ -41,8 +43,49 @@ public void submitTopology(final String topologyName, final Map<?, ?> conf, fina
 
 	public void submitTopologyWithOpts(final String topologyName, final Map<?, ?> conf, final FlinkTopology topology,
 			final SubmitOptions submitOpts) throws Exception {
-		ClusterUtil
-				.startOnMiniCluster(topology.getStreamGraph().getJobGraph(topologyName), topology.getNumberOfTasks());
+		JobGraph jobGraph = topology.getStreamGraph().getJobGraph(topologyName);
+		Configuration jobConfiguration = jobGraph.getJobConfiguration();
+
+		/* storm conf type must be Map<String, Object> */
+		Map<String, Object> stormConf = (Map<String, Object>)conf;
```

Comment:
The configuration in Storm is "public class Config extends HashMap<String, Object>".It extends HashMap<String, Object>,if i use untyped Map, maybe it is hard to convert it into Storm Config.